src/cls/rbd/cls_rbd.cc

```diff
@@ -3097,6 +3097,7 @@ static const std::string IMAGE_KEY_PREFIX("image_");
 static const std::string GLOBAL_KEY_PREFIX("global_");
 static const std::string STATUS_GLOBAL_KEY_PREFIX("status_global_");
 static const std::string INSTANCE_KEY_PREFIX("instance_");
+static const std::string MIRROR_IMAGE_MAP_KEY_PREFIX("map_image_");
```

Comment:
In the sense, I used `instance_map` earlier and that got these keys to be listed out where `INSTANCE_KEY_PREFIX` was being used (leader object). So I stuck to keeping it distinct. `image_map` would be safe to use here.